### DISCUSSION TITLE
Is it possible to clear port level statistics with lldpcli?

### DISCUSSION THREAD

**Discussion by bsun-sudo** [lldpd/lldpd#667] [Q&A]
Hi, 

I'm using lldpcli to display port-level statistics. I would also like to clear these statistics for better debuggability. However, I cannot find a clear statistics command in lldpcli. Is there a way to clear the statistics without altering the LLDP protocol state?

Thanks,
Bing

**Comment by vincentbernat** [comment]
This is currently not implemented.

**Comment by bsun-sudo** [comment]
Thanks for your confirmation.
Bing
